src/middleware/navigation/has.beneficial.owners.or.managing.officers.middleware.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response, NextFunction } from 'express';
+
+import { logger } from '../../utils/logger';
+import { SOLD_LAND_FILTER_URL } from '../../config';
+import { getApplicationData } from "../../utils/application.data";
+import { checkHasNotBOsOrMOs, NavigationErrorMessage } from './check.condition';
+
+export const hasBOsOrMOs = (req: Request, res: Response, next: NextFunction): void => {
```

Comment:
This might be a silly question, but could all these checks be done in a single middleware class? So a bunch of conditions within the 'if' statement, but all resulting in the redirect to the sold land filter page?

Reply:
I thought about it, but the solution is a single function with many `if and else` based on the path route, but to be honest it not really efficient or readable